cmd/samConsensus/samConsensus.go

```diff
@@ -83,23 +79,21 @@ func samConsensus(samFileName string, refFile string, outFile string, vcfFile st
 
 	fmt.Fprintf(outVcfFile, "%s\n", strings.Join(vcf.NewHeader(samFileName).Text, "\n"))
 	var current dna.Base
-	//fmt.Printf("Voting matrix complete, time to vote.\n")
 	var maxList []dna.Base
-	for i = 0; i < len(ref); i++ {
-		for k = 0; k < len(ref[i].Seq); k++ {
+	for i := range ref {
+		for k := range ref[i].Seq {
 			current = voter(votingMatrix[ref[i].Name][k], maxList)
 			if current == dna.N && ref[i].Seq[k] != dna.N {
 				current = dna.ToLower(ref[i].Seq[k])
 			} else if current != ref[i].Seq[k] {
-				fmt.Fprintf(outVcfFile, "%s\t%d\t%s\t%s\t%s\t%s\t%d\t%s\t%s\t%s\t%s\n", ref[i].Name, int64(k+1), "nil", dna.BaseToString(ref[i].Seq[k]), dna.BaseToString(current), "nil", 0, "nil", "SVTYPE=SNP", "nil", "nil")
+				fmt.Fprintf(outVcfFile, "%s\t%d\t%s\t%s\t%s\t%s\t%s\t%s\t%s\n", ref[i].Name, int64(k+1), ".", dna.BaseToString(ref[i].Seq[k]), dna.BaseToString(current), ".", ".", ".", ".")
```

Comment:
What if instead of "." we just have "", since that is also a valid string? Would that create an output file that looks ok and  fits vcf requirements?

Reply:
I mirrored what was in the VCF documentation. I was under the impression it couldn't be blank but let me know if you read something that suggests otherwise.

Reply:
I see! If nothing says we can't have "." then it's probably fine?